### ISSUE TITLE
ProcessException: ProcessException: Process "xcrun" exited abnormally during `xcrun dsymutil build/aot/armv7/App.framework/App -o build/aot/armv7/App.framework.dSYM.noindex`

### ISSUE THREAD

**Issue by yuridebastiani** [flutter/flutter#44440]
Attached de log file.
[flutter_10.log](https://github.com/flutter/flutter/files/3824666/flutter_10.log)

**Comment by iapicca** [comment]
Hi @yuridebastiani 
you might want to take a look at this possible [solution](https://github.com/flutter/flutter/issues/34477#issuecomment-515817558),
let me know if it worked for you
thank you

**Comment by yuridebastiani** [comment]
Hi @iapicca 
Thanks for your help.
Unfortunately this solution don't worked.
I changed the version from 1.0.1 to 1.0.1+1 in pubspec.yaml.
Then I run "flutter clean" and "flutter build ios",  but its finished with the same error.

**Comment by jmagman** [comment]
Can you run with the `--verbose` flag?  I can't see the actual xcrun error message in that log.

**Comment by yuridebastiani** [comment]
I attached the command.

[flutter_build_ios_verbose.txt](https://github.com/flutter/flutter/files/3831372/flutter_build_ios_verbose.txt)

**Comment by yuridebastiani** [comment]
@iapicca any solution?

**Comment by jmagman** [comment]
Annoyingly the verbose logs don't say what the bad exit code was.  However, it looks like you are targeting a simulator in release mode, which isn't supported.  See https://github.com/flutter/flutter/issues/11754.  I definitely would have expected a better error code than this.

Do you see the crash when you either target a simulator or run with `--debug`?

**Comment by yuridebastiani** [comment]
When I run with --debug, the build run normaly. I tested in a Simulator and the App work fine in debug mode.
If I try to build with XCode, I got the same error.
But, if I try to build in Terminal another App, it's works fine. The problem seens to be just with this App.

**Comment by yuridebastiani** [comment]
I have a solution.

First I updated my Mac Os and XCode to the latest version.

Then I create with Android Studio a new Flutter Project, with the same package name of the older project.

Then I copy my lib folder, assets folder, pubspec.yaml and App icons from the older project to the new project.

I got the packages of pubspec.yaml with Android Studio's button "package get".

After this. I close the Android Studio and open the .xcworkspace file in XCode.

Then I setup the signing settings, just it, and close the XCode.

So I run in Terminal (in App folder) `flutter build ios`.
The command run perfectly.

After the command finish, I open again the .xcworkspace file in XCode and setup the project normaly, like, Display Name, Version, App Icon, etc.

Then I build and archive the project with XCode normaly, uploading it to the App Store.

This worked for me.
Thank tou guys for all help.
